scripts/add-cahps-measures.js

```diff
@@ -0,0 +1,91 @@
+// Libraries
+const fs    = require('fs');
+const parse = require('csv-parse');
```

Comment:
Is it ? https://github.com/CMSgov/qpp-measures-data/blob/master/package.json#L16 I think sung used it for his benchmark work so it already exists